README.md

```diff
@@ -12,9 +12,8 @@
 [![Youtube Demo Video][youtube-shield]][youtube-url]
 [![Twitter Follow][twitter-shield]][twitter-url]
 
-### **Tabnine Overview:**
 
-​
+
```

Comment:
delete the extra blank lines, only leave a single blank line (in markdown any 2 blank lines are another paragraph, so this will leave too much blank space